docs/source/data/context.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Context and contextValue
+description: Sharing information and request details throughout your server
+---
+
+During a GraphQL operation, you can share data throughout your server's [resolvers](./resolvers) and [plugins](../builtin-plugins) by creating an object named `contextValue`.
+
+You can pass useful things through your `contextValue` that any resolver might need, like [authentication scope](https://blog.apollographql.com/authorization-in-graphql-452b1c402a9), [sources for fetching data](./fetching-data/), database connections, and custom fetch functions. If you're using [dataloaders to batch requests](./fetching-data/#batching-and-caching) across resolvers, you can also attach them to the shared `contextValue`.
```

Comment:
If we're going to link to that blog post we should probably update it to AS4 (`@apollo/server`)

Reply:
I realized this should probably point to the docs `Auth` article on this topic instead of an old blog post. I've updated this link to point to: https://www.apollographql.com/docs/apollo-server/security/authentication#putting-authenticated-user-info-on-the-context

Reply:
I'm going to merge this in, but let me know if you feel like that blog is a better resource to link to, and I can change the link back and update that article!

Reply:
Looks good